src/redist/targets/BundledManifests.targets

```diff
@@ -0,0 +1,64 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <BundledManifests Include="Microsoft.NET.Workload.Android" Version="$(XamarinAndroidWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.Android" />
+    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.BlazorWebAssembly" />
+  </ItemGroup>
+
+  <!-- Restore workload manifests via PackageReference -->
+  <ItemGroup>
+    <PackageReference Include="@(BundledManifests)" />
+  </ItemGroup>
+
+  <Target Name="LayoutManifests"
+        DependsOnTargets="LayoutManifestsForSDK;LayoutManifestsForMSI" />
+
+  <Target Name="LayoutManifestsForSDK"
+          DependsOnTargets="SetupBundledComponents;GenerateManifestVersions">
+    
+    <Copy SourceFiles="@(ManifestContent)"
+          DestinationFolder="$(RedistLayoutPath)sdk-manifests/$(CliProductBandVersion)00/%(DestinationPath)"/>
+  
+  </Target>
+
+  <Target Name="LayoutManifestsForMSI"
+          DependsOnTargets="SetupBundledComponents;GenerateManifestVersions"
+          Condition="$(ProductMonikerRid.StartsWith('win')) And '$(Architecture)' != 'arm'">
+
+    <Copy SourceFiles="@(ManifestContent)"
+          DestinationFolder="$(BaseOutputPath)$(Configuration)\sdk-manifests\sdk-manifests/$(CliProductBandVersion)00/%(DestinationPath)"/>
+    
+  </Target>
+  
+  <Target Name="GenerateManifestVersions">
+    
+    <PropertyGroup>
+      <ManifestsMSIInstallerFile>$(ArtifactsNonShippingPackagesDir)dotnet-sdkmanifests-$(FullNugetVersion)-$(ProductMonikerRid)$(InstallerExtension)</ManifestsMSIInstallerFile>
```

Comment:
I'm generating one MSI that includes all workload manifests right now. Is this the right approach?

Reply:
I think it's fine for now.  Eventually there will be a separate MSI for each manifest (and for each workload pack), but those won't even be generated in the installer repo.  The tooling that @joeloff is working on will do that generation.